packages/bundler-plugin-core/src/utils/providers/AppVeyorCI.ts

```diff
@@ -107,14 +143,14 @@ export async function getServiceParams(
   output: Output,
 ): Promise<ProviderServiceParams> {
   return {
-    branch: _getBranch(inputs),
-    build: _getBuild(inputs),
-    buildURL: _getBuildURL(inputs),
+    branch: _getBranch(inputs, output),
```

Comment:
I personally found the arg name output a bit confusing since if that is passed as an arg I'd wonder if that's something passed by reference expecting to get mutated.

Fine to leave as is since the pattern is already well-established across the rest of the code, just a passing musing